utils/plots.py

```diff
@@ -23,6 +23,9 @@
 matplotlib.rc('font', **{'size': 11})
 matplotlib.use('Agg')  # for writing to files only
 
+FILE = Path(__file__).absolute()
+ROOT = FILE.parents[1]  # yolov5/ dir
```

Comment:
@glenn-jocher this line breaks any system that doesn't allow writing on this specific directory.

For example, AWS Lambda Functions only allow writing to `/tmp`.

This is an issue because when pulling `git pull ...ultralytics/yolov5` and running it inside a Lambda function, it will fail with:

```
[ERROR] OSError: [Errno 30] Read-only file system: '/var/task/yolov5/tmpgg9rbwwq'
Traceback (most recent call last):
  File "/var/task/pyworker/models/mod/find_objects.py", line 15, in __init__
    self.model = object_detector()
  File "/var/task/pyworker/models/ml_models.py", line 9, in object_detector
    return torch.hub.load('yolov5', 'custom', path=model_path, source='local')
  File "/var/lang/lib/python3.8/site-packages/torch/hub.py", line 364, in load
    model = _load_local(repo_or_dir, model, *args, **kwargs)
  File "/var/lang/lib/python3.8/site-packages/torch/hub.py", line 393, in _load_local
    model = entry(*args, **kwargs)
  File "/var/task/yolov5/hubconf.py", line 70, in custom
    return _create(path, autoshape=autoshape, verbose=verbose, device=device)
  File "/var/task/yolov5/hubconf.py", line 30, in _create
    from models.yolo import Model
  File "/var/task/yolov5/models/yolo.py", line 17, in <module>
    from models.common import *
  File "/var/task/yolov5/models/common.py", line 23, in <module>
    from utils.plots import Annotator, colors
  File "/var/task/yolov5/utils/plots.py", line 64, in <module>
    class Annotator:
  File "/var/task/yolov5/utils/plots.py", line 65, in Annotator
    check_font()  # download TTF if necessary
  File "/var/task/yolov5/utils/plots.py", line 60, in check_font
    torch.hub.download_url_to_file(url, str(font))
  File "/var/lang/lib/python3.8/site-packages/torch/hub.py", line 433, in download_url_to_file
    f = tempfile.NamedTemporaryFile(delete=False, dir=dst_dir)
  File "/var/lang/lib/python3.8/tempfile.py", line 540, in NamedTemporaryFile
    (fd, name) = _mkstemp_inner(dir, prefix, suffix, flags, output_type)
  File "/var/lang/lib/python3.8/tempfile.py", line 250, in _mkstemp_inner
    fd = _os.open(file, flags, 0o600)
```

Is there a way to configure the directory to which `Arial.ttf` is written?